mapApp/static/mapApp/js/index-helpers.js

```diff
@@ -0,0 +1,565 @@
+/** This file contains all the variable and function definitions that are used by index.js but are not directly related to the initalization of the map location.
+ */
+
+/** Host location, port to construct API requests */
+var hostname = window.location.hostname;
+
+// If running locally, include port in API requests
+if (window.location.port) {
+  hostname = hostname + ':' + window.location.port;
+}
+
+/**
+ * Variable declarations for map and data loading
+ */
+let map;
+let mapCenter, mapZoom;
+let alertAreas = L.featureGroup();
```

Comment:
I'm not sure if it could cause an issue or not, but a variable named 'alertAreas' is declared with var in vis.html. Is there any chance of some weird collision happening? Maybe declare this one with const?

Reply:
I updated it for now! I was talking to Trisalyn about getting the vis page map to load the same coordinates of the index page map, so I think the vis page code that loads the map to the alert area bounds will probably be removed soon.